### ISSUE TITLE
Unoptimized prime testing

### ISSUE THREAD

**Issue by maucon** [TheSwerik/CalcPrimesMultiThread#6]
CalcPrimesMultiThread/Backend/src/Prime/Task/TaskMaster.cs 
Starting at line 47
```
for (BigInteger i = 0;
                 current < Max && !(token?.IsCancellationRequested ?? false);
                 current = lastPrime + ++i * Step)
{
...
}
```
Why going through every number? Just check uneven numbers -> so for-step += 2

CalcPrimesMultiThread/Backend/src/Prime/Task/StaticPrime.cs 
Line 12
`if ((n & 1) == 0) return false;`

Can be deleted then.

**Comment by TheSwerik** [comment]
Because its not that easy to change that to two stept, need to rethink the entire class for that

**Comment by TheSwerik** [comment]
fixed in 0ddd25c

.. i think

**Comment by maucon** [comment]
CalcPrimesMultiThread/Backend/src/Prime/Task/StaticPrime.cs
Line 12
`if ((n & 1) == 0) return false;`

Can be deleted...

**Comment by TheSwerik** [comment]
fixed in 05c96dc
